src/System.Management.Automation/engine/CultureVariable.cs

```diff
@@ -25,7 +25,7 @@ public override object Value
             get
```

Comment:
@iSazonov, your last commit had 1 failures in `PowerShell-CI-macos`
$PSCulture.$PSCulture follows the current thread culture
```powershell
Expected strings to be the same, but they were different.
String lengths are both 5.
Strings differ at index 0.
Expected: 'ru-RU'
But was:  'en-US'
at <ScriptBlock>, /Users/vsts/agent/2.153.2/work/1/s/test/powershell/Modules/Microsoft.PowerShell.Utility/Get-Culture.Tests.ps1: line 77
77:             $PSCulture | Should -BeExactly 'ru-RU'
```